repose/argparsing.py

```diff
@@ -121,115 +121,115 @@ def get_parser():
 
     cmdremove = commands.add_parser(
         'remove',
-        help='Removes repositories from targets')
+        help='remove repository from target')
```

Comment:
i thing this can remove more repositories from refhost

Reply:
Sure, and from multiple targets. I decided to keep it in singular form as it should be obvious, that you can add multiple targets and repository patterns.